StellarApp/Controllers/NewsView.swift

```diff
@@ -31,18 +31,17 @@ class NewsView: UIView {
     }
     
     private func commonInit() {
-    
         setConstraints()
     }
     
     func setConstraints() {
-        self.addSubview(NewsCollectionView)
-        NewsCollectionView.backgroundColor = .clear
-        NewsCollectionView.translatesAutoresizingMaskIntoConstraints = false
-        NewsCollectionView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: 11).isActive = true
-        NewsCollectionView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor, constant: 8).isActive = true
-        NewsCollectionView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -8).isActive = true
-        NewsCollectionView.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor, constant: -11).isActive = true
+        self.addSubview(newsCollectionView)
+        newsCollectionView.backgroundColor = .clear
+        newsCollectionView.translatesAutoresizingMaskIntoConstraints = false
+        newsCollectionView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor, constant: 11).isActive = true
```

Comment:
do you need the space around the collection view or can it just be 0 all around?